app/Helpers/MailSend.php

```diff
@@ -26,7 +26,10 @@ public static function send($view, $data, $profile, $subject)
                 $subject
             ) {
                 $message->from($emailFrom, $emailName);
-                $message->to($profile->email, $profile->name)->subject($emailName . ' - ' . $subject);
+                if (is_object($profile)) {
```

Comment:
This is bad approach - all should be instances of GenericModel (with `profiles` collection)